sim/paladin/talents.go

```diff
@@ -195,7 +195,7 @@ func (paladin *Paladin) applyReckoning() {
 				ActionID:    actionID,
 				SpellSchool: core.SpellSchoolPhysical,
 				Flags:       core.SpellFlagMeleeMetrics,
-
+				ProcMask:    core.ProcMaskMeleeMH,
```

Comment:
For consistency, can we put `ProcMask` after `SpellSchool` and before `Flags` like everywhere else?